From: =?utf-8?b?SMOpY3RvciBPcsOzbiBNYXJ0w61uZXo=?=
 <hector.oron@collabora.co.uk>
Date: Mon, 6 Nov 2017 15:55:02 +0100
Subject: debootstrap: add fallback for Debian SID distro
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: base64

ICBXaGVuIGF0dGVtcHRpbmcgdG8gYnVpbGQgRGViaWFuIHVuc3RhYmxlLCBkaXN0cmlidXRpb24g
aXMKICBjYWxjdWxhdGVkIHVzaW5nIGxzYiwgd2hpY2ggcmVsaWVzIG9uIGFwdCBjYWNoZS4gQXMg
b3VyCiAgYnVpbGQgc3lzdGVtcyBsYWNrIGFwdCBjYWNoZSBhbmQgbmV0d29yayB3ZSBjYW5ub3Qg
ZGV0ZWN0CiAgd2UgYXJlIGJ1aWxkaW5nIGZvciBEZWJpYW4gU0lEIChha2EgdW5zdGFibGUpLgog
IEZvciBtb3JlIGJhY2tncm91bmQgaW5mb3JtYXRpb24gc2VlOiBEZWJpYW5CdWcjODQ1NjUxCgpT
aWduZWQtb2ZmLWJ5OiBIw6ljdG9yIE9yw7NuIE1hcnTDrW5leiA8aGVjdG9yLm9yb25AY29sbGFi
b3JhLmNvLnVrPgpUZXN0ZWQtYnk6IEjDqWN0b3IgT3LDs24gTWFydMOtbmV6IDxoZWN0b3Iub3Jv
bkBjb2xsYWJvcmEuY28udWs+Cg==
---
 build-recipe-debootstrap | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/build-recipe-debootstrap b/build-recipe-debootstrap
index 72fb599..c06ef34 100644
--- a/build-recipe-debootstrap
+++ b/build-recipe-debootstrap
@@ -31,7 +31,11 @@ recipe_prepare_debootstrap() {
 
 recipe_build_debootstrap() {
     local arch=$(chroot $BUILD_ROOT su -c "dpkg-architecture -qDEB_BUILD_ARCH")
-    local dist=$(chroot $BUILD_ROOT su -c "lsb_release --codename --short")
+    if $(chroot $BUILD_ROOT grep -q '/sid' /etc/debian_version) ; then
+        local dist=unstable
+    else
+        local dist=$(chroot $BUILD_ROOT su -c "lsb_release --codename --short")
+    fi
     local myroot=debootstraproot
     test -d $BUILD_ROOT/.build.binaries || cleanup_and_exit 1
     if test "$DO_INIT" = true -o ! -d "$BUILD_ROOT/.build.binaries/dists" ; then
